### DISCUSSION TITLE
Will users customize their own scorecard?

### DISCUSSION THREAD

**Discussion by randytpierce** [NOAA-GSL/MATS#777] [Scorecard - user discussions]
Question posed by Steve W during meeting on Aug 26, 2022:
Is it correct that users will “customize” their own “scorecard” in an analogous way to how they currently “customize” a single verification plot request?

**Comment by randytpierce** [comment]
Answer from Randy:
Absolutely - if I understand the question correctly. We can visualize this on the demonstration of the UI

**Comment by randytpierce** [comment]
If 2. Is true, I have the following questions / concerns:
Will the “system” be able to “handle” all the different scorecards?
answer: The backend data for the scorecards will be couchbase. I (Randy) have confidence that we can handle all the data. The problem will be the processing and we have a few ideas for that. Unfortunately I think experience is going to have to teach us where the processing limits are.

**Comment by randytpierce** [comment]
[Tech Q]:  Is the scorecard envisioned to just compare two different model systems (retro or R/T?  I (Steve) am assuming so (too many DOFs to do anything else).  – [thanks for discussion on this].
answer: at the moment only two. a “data source” and a “validation data source”

**Comment by randytpierce** [comment]
[Tech Q]: What might be the “minimal requirements” for a given R/T or retro system to be “eligible” for scorecard verif?  It seems like lots of “logistical matching” issues to be dealt with here – need matched period, domain, etc.  (all the things that are already handled now) PLUS a matched set of underlying verifications (UA, SFC, REFL, CEILING, etc.]  
answer: I (Randy) think this needs more discussion and perhaps may have to wait for the prototype to provide an example for discussion. Unless Molly or Dave want to take a stab at it.

**Comment by bonnystrong** [comment]
Follow-on from Steve W:
Assuming so, we will likely end up with multiple (many) different “customized” regional and global “scorecards” (kind of analogous to an AWIPS display “procedure”?

**Comment by bonnystrong** [comment]
Answer from Randyto question about ending up with multiple different cutomized versions:
Yes

**Comment by bonnystrong** [comment]
Another follow-on from Steve:

Will the “system” be able to “handle” all the different scorecards?

**Comment by bonnystrong** [comment]
Answer from Randy:

The backend data for the scorecards will be couchbase. I have confidence that we can handle all the data. The problem will be the processing and we have a few ideas for that. Unfortunately I think experience is going to have to teach us where the processing limits are.
